jwst/jump/Tests/test_twopoint_difference.py

```diff
@@ -0,0 +1,431 @@
+
+from twopoint_difference import find_CRs
```

Comment:
In the jwst package we need imports to be relative, which means this needs to be:

from ..twopoint_difference import find_CRs

to get the import to succeed. Right now it's failing in the Travis tests.